src/scripts/windows/set-gitcredential.sh

```
@@ -0,0 +1,53 @@
+#!/bin/false
+# shellcheck shell=bash
+# shellcheck disable=SC2016,SC2154
+
+readonly base_dir="${CIRCLE_WORKING_DIRECTORY/\~/$HOME}"
+readonly unity_project_full_path="$base_dir/$PARAM_PROJECT_PATH"
```

Shellcheck (0.9.0-1)

[warning] unity_project_full_path appears unused. Verify use (or export if used externally).

(SC2034)